src/chainstate/coordinator/tests.rs

```diff
@@ -3401,14 +3462,32 @@ fn test_sbtc_peg_in_btc_op() {
     }
 
     let total_number_of_peg_in_ops = burnchain_block_hashes
-        .into_iter()
+        .iter()
         .flat_map(|block_hash| {
-            SortitionDB::get_peg_in_ops(&sort_db.conn(), &block_hash)
+            SortitionDB::get_peg_in_ops(&sort_db.conn(), block_hash)
                 .expect("Failed to get peg in ops")
         })
         .count();
 
+    let total_number_of_peg_out_request_ops = burnchain_block_hashes
+        .iter()
+        .flat_map(|block_hash| {
+            SortitionDB::get_peg_out_request_ops(&sort_db.conn(), block_hash)
+                .expect("Failed to get peg out request ops")
+        })
+        .count();
+
+    let total_number_of_peg_out_fulfill_ops = burnchain_block_hashes
+        .iter()
+        .flat_map(|block_hash| {
+            SortitionDB::get_peg_out_fulfill_ops(&sort_db.conn(), block_hash)
+                .expect("Failed to get peg out fulfillment ops")
+        })
+        .count();
+
     assert_eq!(total_number_of_peg_in_ops, 1);
+    assert_eq!(total_number_of_peg_out_request_ops, 1);
+    assert_eq!(total_number_of_peg_out_fulfill_ops, 2);
```

Comment:
Please also verify the contents of these operations.

Reply:
Done.